$NetBSD: patch-hscolour.cabal,v 1.1 2021/04/24 17:00:46 pho Exp $

Fix build with GHC 9. This is probably a Cabal bug. When a package
defines both a library an executable which share the same set of
modules, Cabal appears to link an executable with rpath pointing at
the build directory which doesn't serve any purpose.

--- hscolour.cabal.orig	2021-04-24 16:56:21.123893246 +0000
+++ hscolour.cabal
@@ -20,7 +20,7 @@ Description:
 Category: Language
 Build-Type: Simple
 Data-files: hscolour.css, data/rgb24-example-.hscolour
-Cabal-version: >=1.6
+Cabal-version: >=1.8
 
 
 Library
@@ -47,7 +47,7 @@ Library
 
 
 Executable HsColour
-  Build-depends: base < 10, containers
+  Build-depends: base < 10, containers, hscolour
   Main-is: HsColour.hs
   --ghc-options: -O -W
   Extensions: CPP
